script.js

```diff
@@ -0,0 +1,172 @@
+import {
+  keysCode, en, enCaps, ru, ruCaps, controlsKey,
+} from './assets/data.js';
+
+const body = document.querySelector('body');
+const wrapper = document.createElement('div');
+const textarea = document.createElement('textarea');
+const keyboard = document.createElement('div');
+const hint = document.createElement('div');
+
+
+body.appendChild(wrapper);
+wrapper.classList.add('wrapper');
+wrapper.appendChild(hint);
+hint.classList.add('hint');
+hint.innerHTML = 'Переключение языка: Ctrl + Alt <br> Клавиатура создана в OS Windows';
+wrapper.appendChild(textarea);
+textarea.classList.add('textarea');
+wrapper.appendChild(keyboard);
+keyboard.classList.add('keyboard-wrapper');
+
+let caps = false;
+let langEn = JSON.parse(localStorage.getItem('langEn'));
+
+function keyboardKey() {
+  let layout = langEn;
+  if (layout) {
+    if (caps) {
+      layout = enCaps;
+    } else {
+      layout = en;
+    }
+  } else if (caps) {
+    layout = ruCaps;
+  } else {
+    layout = ru;
+  }
+  keyboard.querySelectorAll('span').forEach((item) => item.remove());
+
+  for (let i = 0; i < keysCode.length; i += 1) {
+    let keyClass = '';
+    if (keysCode[i] === 'Tab' || keysCode[i] === 'Delete' || keysCode[i] === 'ControlLeft' || keysCode[i] === 'ControlRight' || keysCode[i] === 'MetaLeft' || keysCode[i] === 'AltLeft' || keysCode[i] === 'AltRight') {
+      keyClass = `control ${keysCode[i].toLowerCase()}`;
+    }
+    if (keysCode[i] === 'Backspace' || keysCode[i] === 'CapsLock' || keysCode[i] === 'Enter' || keysCode[i] === 'ShiftLeft' || keysCode[i] === 'ShiftRight') {
+      keyClass = `control middle ${keysCode[i].toLowerCase()}`;
+    }
+    if (keysCode[i] === 'Space') {
+      keyClass = 'control space';
+    }
+    keyboard.innerHTML += `<span class="key ${keyClass}" data-code=${keysCode[i]}>${layout[i]}</span>`;
+  }
+}
+
+function changeKeyboard() {
+  let layout = langEn;
+  if (layout) {
+    if (caps) {
+      layout = enCaps;
+    } else {
+      layout = en;
+    }
+  } else if (caps) {
+    layout = ruCaps;
+  } else {
+    layout = ru;
+  }
+
+  keyboard.querySelectorAll('span').forEach((item, i) => {
+    const span = item;
+    span.innerText = layout[i];
```

Comment:
I guess you don't need the var `span`. You could directly set `innerText` to the `item`